lua/cfc_block_luarun/config.lua

```diff
@@ -0,0 +1,32 @@
+local ALLOWED_LUA = {
+	"f3a9e5157f25d221d2b3387f55cc22c6", -- gm_excess_island_night
+	"8f830f5939d09b7cfee4851d4132e571", -- gm_excess_island_night
+	"377582bb92686b70a4a7377b1de6f3a3", -- gm_excess_island_night
+	"e6073b3b67df761aba65074226bab173", -- gm_excess_island_night
+	"c18fdfcdbfb678350b2051f009ad2a16", -- gm_excess_island_night
+	"42aa5a2b7591fb638b2fcfcb37576863", -- gm_excess_island_night
+	"256507181f03abc65f1345fbcc34b3c7", -- gm_excess_island_night
+	"873f7d587c3293aa2f93365f9fddb8ed", -- gm_excess_island_night
+	"8ffa1646c8c427faecfaf353337c7618", -- gm_excess_island_night
+	"7968b1850ee7892d5dc52b6939a364fa", -- gm_excess_island_night
+	"7da014290063534942cfb9ad31fae513", -- gm_excess_island_night
+	"8eb4db77605aa51d562828f2b7f24736", -- gm_excess_island_night
+	"561c2762ab141eb25d13cd9c02a014d4", -- gm_excess_island_night
+	"8ab9987a343192dd663914e820dea337", -- gm_excess_island_night
+	"5cdbbc8b887c769ed18889aed644bc2f", -- gm_excess_island_night
+	"51869559a9ecdc08761315f415542f4d", -- gm_excess_island_night
+	"e43e380cd851f7ea07b7f00ef15cd043", -- gm_excess_island_night
+	"58928d8a6a2defb4d9ce1dd9ef591d05", -- gm_excess_island_night
+	"59364c0e5e56b6a71012bcda890b27ee", -- gm_excess_island_night
+	"bae837e43fb6f9de87b70492b99d4672", -- gm_excess_island_night
+	"2422fe3842a6ed147f024b0220291d46", -- gm_excess_island_night
+	"7db1f3e9eb761fe485b0a4f7e6c2d23b", -- gm_excess_island_night
+	"331ccbeabd5a035a7bccb0ddf7a27501", -- gm_excess_island_night
+	"6ed0feab7ba45c2b96615af64150db5a", -- gm_excess_island_night
+}
+
+local ALLOWED_MAPS = {
+	"gm_excess_island_night",
```

Comment:
it'd be better to do a keyed table
like:
```lua
["gm_excess_island_night"] = true,
```

Reply:
👍